app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -43,21 +44,34 @@
 
     SharedPreferences sharedpreferences;
     SharedPreferences.Editor editor;
+    private Button okButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_trustees);
 
-        System.out.println(comradeEditText.size());
-        comradeEditText.add((EditText) findViewById(R.id.comrade1EditText));
```

Comment:
If it is necessary to make changes in the future, iterating list is easy and consistent than making individual changes. In optimization point of view the previous code is much better than the newer one when considering the overhead of creating local variable and adding it to the list happen in two executions